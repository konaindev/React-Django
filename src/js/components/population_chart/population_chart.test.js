import renderer from "react-test-renderer";
import PopulationChart from "./index";

describe("PopulationChart", () => {
  it("renders correctly", () => {
    const props = {
      segment_population: 10368,
      group_population: 3188
    };

    const tree = renderer.create(<PopulationChart {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
