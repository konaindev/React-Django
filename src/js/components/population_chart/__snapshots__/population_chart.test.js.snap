// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopulationChart renders correctly 1`] = `
<div
  className="population-chart"
>
  <div
    className="population-chart__labels"
  >
    <div
      className="population-chart__label-start"
    >
      0
    </div>
    <div
      className="population-chart__label-end"
    >
      10,368
    </div>
    <div
      className="population-chart__label-value"
      style={
        Object {
          "left": "30.748456790123456%",
        }
      }
    >
      3,188
    </div>
  </div>
  <div
    className="population-chart__bar"
  >
    <div
      className="population-chart__bar-percent"
      style={
        Object {
          "width": "30.748456790123456%",
        }
      }
    />
  </div>
  <div
    className="population-chart__dotted-lines"
  >
    <div
      className="population-chart__dotted-diagnoal"
      style={
        Object {
          "left": "30.748456790123456%",
        }
      }
    >
      <svg
        height="40"
        preserveAspectRatio="none"
        viewBox="0 0 100 40"
        width="100%"
      >
        <path
          d="M0 0 L100 40"
          stroke="#2c3646"
          strokeDasharray="1"
        />
      </svg>
    </div>
  </div>
</div>
`;
